.github/actions/free-disk-space/action.yml

```diff
@@ -0,0 +1,59 @@
+name: "Free Disk Space"
+description: "Remove unneeded preinstalled Docker images and software to free disk space."
+author: "The ORT Server Authors"
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Print Disk Space
+      shell: bash
+      run: df -h
+    - name: List Docker Images
+      if: ${{ false }} # Can be enabled if the 'Remove Unneeded Docker Images' step below needs to be updated.
+      shell: bash
+      run: docker images
+    - name: Remove Unneeded Docker Images
+      shell: bash
+      run: |
```

Comment:
Is clearing some of this an unoptimization ? If for instance ubuntu:22.04 is required for build as  base image, it now has to be downloaded everytime ?

Reply:
If we would use any of those images, yes, but I don't think we ever will.